thirdparty/pip/requirements.in

```diff
@@ -1,3 +1,5 @@
+azure-mgmt-resource
```

Comment:
This file is intended to be the "public" packages that `rules_pip` depends on and exposes via macros/rules.

Because we're using this for internal testing within the repo I suggest we create `thirdparty/pip/internal/requirements.in` and create `@pip3_internal`

Reply:
Agreed, we should not need to pull in these packages in dependent repos if they don't need them (e.g. i95)

Reply:
Ahh I didn't realize that depending packages would use these packages, I thought it was just for the unit testing. I'll update